time-calculator/time_calculator.py

```diff
@@ -0,0 +1,67 @@
+
+def add_time(startTime, stopTime, dayOfWeek=None):
+
+    # Start Time
```

Comment:
Extra comment and extra blank line above .... this should be avoided based on feedback on previous PR's